crates/web-sys/src/features/gen_AnimationPropertyDetails.rs

```diff
@@ -63,6 +63,8 @@ impl AnimationPropertyDetails {
     #[doc = "Change the `values` field of this object."]
     #[doc = ""]
     #[doc = "*This API requires the following crate features to be activated: `AnimationPropertyDetails`*"]
+    #[doc = ""]
+    #[doc = "While the iterable or array can produce any JsValue as far as the type system is concerned, practically it is expected to contain a <code>&[AnimationPropertyValueDetails]</code>."]
```

Comment:
I think it'd be good to generate a link to `AnimationPropertyValueDetails` here? Or does that actually already happen due to markdown details?

Reply:
This is a link -- the brackets make this a link (it's just that in source code it looks like a slice; it isn't).